backend/src/main/resources/application-azure-dev4.yaml

```diff
@@ -15,3 +15,6 @@ simple-report:
       - https://dev4.simplereport.gov
 twilio:
   enabled: true
+  features:
```

Comment:
I think we want to leave this out in this PR and enable it specifically when we need to. Unless I am missing some context? 🤔

Reply:
@emyl3 I did this so I could smoketest that the flag was working in staging. I tried turning the flag on via our endpoint in postman but it wasn't working yesterday. So I decided to turn it on in the config file. I can take it out if you'd like.

Reply:
Also just tried with the endpoint again and it's working today. Idk what is going on with that -- it's the exact same request. Maybe the end point only works to change the value of the flags but not to initialize them in the first part?

Reply:
looks like it should create it if it doesn't exist 🤔 https://github.com/CDCgov/prime-simplereport/blob/7940fbbc456b714d3425676abe5c43b0e5d9ccfe/backend/src/main/java/gov/cdc/usds/simplereport/api/featureflags/FeatureFlagsMutationResolver.java#L20